### PR TITLE
Pull request for #106: Pip 10 issues, switch to using requirements.txt for config_python.py

### PR DESCRIPTION
**Pull request by stanfordquan**
@DavidRitzwoller please review this task. This solution was proposed in [this comment](https://github.com/gslab-econ/template/issues/106#issuecomment-392388229). Please make sure that:

1) the code is up to GSLab standard regarding documentation and style guide (PEP-8). It should also be intuitive for the general user.

2) The fix works for the general user. Please double-check that this should work for older and newer versions of `pip` and for the common Python installations (system Python and Anaconda Python, e.g.). If you have issues here because of your computer's Python please let me know. 

Thanks! Let me know if you have any question.

### CODE REVIEW

**Comment by stanfordquan** [comment]
@DavidRitzwoller can you please prioritize this PR today? This is breaking any fresh clone of template, so the earlier we can get this done the better.

@gentzkow @jmshapir please let me know if you disagree with this prioritizing. Thanks!

**Comment by stanfordquan** [comment]
As part of this PR we will remove the `config_python.py` file and just run the pip install line directly

**Comment by gentzkow** [comment]
Note: For issues like this that we expect to affect current repos based on the template (e.g., ad-price-drivers), we should think about how we want to inform users of those other repos so they can make the update when appropriate.

**Comment by DavidRitzwoller** [comment]
@stanfordquan I've committed changes that fix this issue and the issue raised [here](https://github.com/gslab-econ/gslab_python/issues/154).

File: config/config_python.py [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+    
+    # Install base python requirements, if they are not installed or are the wrong version
+    command = [sys.executable, '-m', 'pip', 'install', '-r', filename]
+    if upgrade:
+        command.append('--upgrade')
+    subprocess.check_call(command)
+    return None
+
+    # Install gslab_tools
+    subprocess.call(['pip', 'install', "git+https://git@github.com/gslab-econ/gslab_python.git@master"])
```
Comment (stanfordquan) [inline config/config_python.py]: this is part of the function, no? so return None should be below line 16

Comment (stanfordquan) [reply config/config_python.py]: I think I asked you this already, but is there any difference between this line and adding 

`git+https://git@github.com/gslab-econ/gslab_python.git@master`

to `requirements.txt`

Comment (DavidRitzwoller) [reply config/config_python.py]: yes you're right

Comment (DavidRitzwoller) [reply config/config_python.py]: Fixed

**Comment by DavidRitzwoller** [comment]
@stanfordquan I made the following updates to Template.
- I removed the config_python.py file in favor of only using the requirements.txt file.
- I updated the README to reflect this change in the installation procedure.

**Comment by stanfordquan** [comment]
# Template 4.1.2 Release Notes

## Bugs

- Pip v.10 update broke our installation method in `template`. We changed our suggested installation method by removing `config_python.py` and suggesting user call `pip install -r config/requirements.txt` directly . (#106 #107)

**Pull request merged**